gammapy/estimators/map/tests/test_asmooth.py

```diff
@@ -75,9 +75,9 @@ def test_asmooth_dataset(input_dataset):
 
     desired = {
         "counts": 369.479167,
-        "background": 0.184005,
+        "background": 0.141492,
         "scale": 0.056419,
-        "sqrt_ts": 72.971513,
+        "sqrt_ts": 74.407912,
```

Comment:
What is happening here? Why should the test values change?

Reply:
It seems the `norm` parameter is not read correctly:
```python3
from gammapy.modeling.models import Models

models = Models.read("$GAMMAPY_DATA/fermi-3fhl-crab/Fermi-LAT-3FHL_models.yaml")
print(models)
```
Before:
```
Models

Component 0: SkyModel

  Name                      : Crab Nebula
  Datasets names            : None
  Spectral model type       : LogParabolaSpectralModel
  Spatial  model type       : PointSpatialModel
  Temporal model type       : 
  Parameters:
    amplitude               :   1.82e-02   +/- 3.0e-03 1 / (cm2 s TeV)
    reference    (frozen)   :      0.000       TeV         
    alpha                   :      1.652   +/-    0.06             
    beta                    :      0.039   +/-    0.01             
    lon_0        (frozen)   :     83.633       deg         
    lat_0        (frozen)   :     22.020       deg         

Component 1: FoVBackgroundModel

  Name                      : Fermi-LAT-bkg
  Datasets names            : ['Fermi-LAT']
  Spectral model type       : PowerLawNormSpectralModel
  Parameters:
    norm                    :      1.300   +/-    0.08             
    tilt         (frozen)   :      0.000                   
    reference    (frozen)   :      1.000       TeV         

```
Can you please check whether this is intended?

After:
```
Models

Component 0: SkyModel

  Name                      : Crab Nebula
  Datasets names            : None
  Spectral model type       : LogParabolaSpectralModel
  Spatial  model type       : PointSpatialModel
  Temporal model type       : 
  Parameters:
    amplitude               :   1.82e-02   +/- 3.0e-03 1 / (cm2 s TeV)
    reference    (frozen)   :      0.000       TeV         
    alpha                   :      1.652   +/-    0.06             
    beta                    :      0.039   +/-    0.01             
    lon_0        (frozen)   :     83.633       deg         
    lat_0        (frozen)   :     22.020       deg         

Component 1: FoVBackgroundModel

  Name                      : Fermi-LAT-bkg
  Datasets names            : ['Fermi-LAT']
  Spectral model type       : PowerLawNormSpectralModel
  Parameters:
    norm                    :      1.000   +/-    0.00             
    tilt         (frozen)   :      0.000                   
    reference    (frozen)   :      1.000       TeV         

```

Reply:
I renamed the model entry `norm_spectral_model` so the files need to be updated $GAMMAPY_DATA
Alternatively I could keep `spectral_model` but test first if 'type' is in data in ._get_model_class_from_dict()

Reply:
changed back  to previous naming